packages/dart/sshnp_flutter/lib/src/utility/sizes.dart

```diff
@@ -60,3 +60,5 @@ const gapH30 = SizedBox(height: Sizes.p30);
 const gapH36 = SizedBox(height: Sizes.p36);
 const gapH46 = SizedBox(height: Sizes.p46);
 const gapH60 = SizedBox(height: Sizes.p60);
+
+const kMinimumWindowSize = Size(671, 505);
```

Comment:
From context of this commit, it looks like this variable is used only for Windows platform; however the variable name doesn't tell me that it is used only for Windows platform ... I'd suggest renaming to `kWindowsMinWindowSize` or at least adding a code comment

Reply:
You're correct, The size for Mac OS was set in Xcode. I'll make the change